src/observer/ingest_logic.py

```diff
@@ -560,6 +584,14 @@ def _(v):
     models.FEATURE: {'description': FEATURE_DESC,
                      'model': models.Content},
 
+    models.EDITORIAL: {'description': EDITORIAL_DESC,
+                       'model': models.Content},
+
+    models.PODCAST: {'description': PODCAST_DESC,
+                     'model': models.Content,
+                     'api-list': 'podcast-episodes',
+                     'idfn': lambda data: str(data['number'])},
```

Comment:
convert to string in desc

Reply:
actually, this is used to index the *raw* data rather than processed stuff. leaving as-is